client/src/components/Auth/PrivateRoute/__tests__/PrivateRoute.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { MemoryRouter } from "react-router";
+import { Redirect } from "react-router-dom";
+import { mount } from "enzyme";
+import { findByTestAttr } from "../../../../test/testUtils";
+import { GenericComponent } from "../../../../test/__mocks__/fileMock";
+import PrivateRoute from "../PrivateRoute";
+import configureStore from "redux-mock-store";
+
+const mockStore = configureStore();
+
+const loggedInState = {
+  auth: {
+    isAuthenticated: true
+  }
+};
+
+const loggedOutState = {
+  auth: {
+    isAuthenticated: false
+  }
+};
+
+describe("PrivateRoute rendering", () => {
+  test("PrivateRoute should render a component if user is logged in", () => {
```

Comment:
No need to repeat `PrivateRoute` in this test description as it's already within a `describe` block with that text.